cta/cta.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
The indentation here could be better to make it more readable. Here's how to set up your Visual Stuido Code to make it better in terms of readability. 

1. Open VS Code
2. Search for `Prettier` in extensions and install it.. 
![image](https://user-images.githubusercontent.com/71005514/182471035-203eea3b-649f-412b-92cd-12d9065329e7.png)

3.  Open settings (you can use the keyboard shortcut `Ctrl + ,`)
4. Search for `formatter` and make sure prettier is enabled as the default formatter
![image](https://user-images.githubusercontent.com/71005514/182471507-486f92c7-70cd-4e72-969a-a0aec9129d7f.png)

5. Make sure that `Format on Paste` and `Format on Save` are checked. 
![image](https://user-images.githubusercontent.com/71005514/182471814-4caebb5c-c8fa-4890-a5da-002b279ccda1.png)

6. Save all your files again (i.e. open them and press `Ctrl + S`) and watch them get formatted properly.